content/guides/public-policy/_index.md

```diff
@@ -75,9 +75,9 @@ Keep reading to learn more about the different [types of policies](https://digit
 
 ---
 
-<footer>
+<div class="dg-footnote">
 <h3 id="footnote-label">Footnotes</h3>
```

Comment:
Is it possible to BEM-ify this `dg-footnote` into a component for the `h3`, `ol` and `li` elements?